File: clientside/src/app/entities/safetydocs/safetydoc.ts
export class Safetydoc {
    id: number;
    status: string;
    type: string;
    switchingPlan: string;
    safetyDocType: string;
    dateCreated: Date;
    phoneNo: string;
    crew: string;
    creator: string;
    details: string;
    notes: string;

    //list devices
    workOperationsCompleted: boolean;
    tagsRemoved: boolean;
    groundingRemoved: boolean;
    readyForService: boolean

    constructor(status: string, type: string, switchingPlan: string, safetyDocType: string,
        dateCreated: Date, phoneNo: string, crew: string, creator: string, details: string, notes: string,
        workOperationsCompleted: boolean, tagsRemoved: boolean, groundingRemoved: boolean, readyForService: boolean){
            this.status = status;
            this.type = type;
            this.switchingPlan = switchingPlan;
            this.safetyDocType = safetyDocType;
            this.dateCreated = dateCreated;
            this.phoneNo = phoneNo;
            this.crew = crew;
            this.creator = creator;
            this.details = details;
            this.notes = notes;

            this.workOperationsCompleted = workOperationsCompleted;
            this.tagsRemoved = tagsRemoved;
            this.groundingRemoved = groundingRemoved;
            this.readyForService = readyForService;
    }
}

File: Core/SmartGrid/Models/Safetydoc.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace SmartGrid.Models
{
    public class Safetydoc
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Status { get; set; }
        public string Type { get; set; }
        public string SwitchingPlan { get; set; }
        public string SafetyDocType { get; set; }
        public DateTime DateCreated { get; set; }
        public string PhoneNo { get; set; }
        public string Crew { get; set; }
        public string Creator { get; set; }
        public string Details { get; set; }
        public string Notes { get; set; }
        public bool WorkOperationsCompleted { get; set; }
        public bool TagsRemoved { get; set; }
        public bool GroundingRemoved { get; set; }
        public bool ReadyForService { get; set; }
    }
}
